src/constants/fieldCategories.js

```diff
@@ -26,7 +26,7 @@ export const defaultSightingCategories = {
 
 export const defaultEncounterCategories = {
   animal: {
-    name: 'individual',
+    name: 'individualInformation',
```

Comment:
mediocre names were causing console warnings for duplicate keys

Reply:
It doesn't look like 'individual' was causing duplicate keys. I agree with updating it for consistency with the other categories though.